Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+    
+    init(JSON: Dictionary<String, AnyObject>) {
```

Comment:
Okay, let's use Argo then.

Hmm, I don't see any advantages of using `NullDailyForecast` - maybe because I never used this pattern. 

Instead of creating nothing we're creating null object - this object means that something goes wrong. 

What's next? Do we create `TRDailyForecastViewModel` with this null object and assign this model to `TRDailyForecastView.model`? How the view handle this? It will change its state internally? Meaning, does this view hide all subviews or itself?

This null object for know is for me the same like:

    @objc class DailyForecast: NSObject {
        let date: NSDate?
        let conditionsDescription: String?
        let highTemperature: Temperature?
        let lowTemperature: Temperature?
        
        lazy var isValid: Bool = {
            return self.date != nil && self.conditionsDescription != nil && self.highTemperature != nil && self.lowTemperature != nil
        }()
    ....